streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamProcessorApiTest.java

```diff
@@ -46,7 +46,7 @@
 import static java.util.Arrays.asList;
 
 
-public class KStreamNewProcessorApiTest {
+public class KStreamProcessorApiTest {
```

Comment:
This is a similar update -- we don't have the old Processor API any longer, so dropping the `New` from this name, too.